src/__configuration__/themes/christmas.ts

```diff
@@ -0,0 +1,62 @@
+import * as PXBThemes from '@pxblue/react-themes';
+import * as Colors from '@pxblue/colors';
+import * as BrandingColors from '@pxblue/colors-branding';
+import ChristmasEve from '../../app/assets/themes/christmas-eve.png';
+import AppBarTile from '../../app/assets/themes/christmas-appbar-tile.png';
+import { Schedule } from './types';
+
+export const ChristmasSchedule: Schedule = {
```

Comment:
Hate to be this guy, but if we're going to do a "Christmas" theme instead of a generic 'holiday' or 'winter' theme, then we're probably also going to need a Kwanzaa theme and a Hanukkah theme.

Reply:
If there is time, I'm sure our counterparts in Pune would be happy to see a Diwali theme in mid November too

Reply:
We don't have to do all of these right now though. But we should have some backlog items to do them in time for their respective holidays.

Reply:
>  I'm sure our counterparts in Pune would be happy to see a Diwali theme in mid November too

Eeeek, I want to just pay someone to do us a banner illustration for that

Reply:
> a Kwanzaa theme and a Hanukkah theme.

I... am not even aware of these holidays. 🤦‍♀️  They will go to the backlog for sure, and let someone who knows the culture better handle it lest I offend anyone with that Vulcan / Star Trek thing.

Reply:
Sounds good, as long as we have a story for them we should be good

Reply:
PXBLUE-1674, listed as priority items to groom for next sprint.